app/build.gradle

```diff
@@ -171,15 +171,15 @@ dependencies {
     String okHttpVersion = '4.12.0'
     String retrofitVersion = '2.9.0'
     String glideVersion = '4.16.0'
-    String mockitoVersion = '5.2.0'
+    String mockitoVersion = '5.10.0'
     String leakCanaryVersion = '2.13'
     String kotlinCoroutinesVersion = '1.7.3'
     String firebaseMessagingVersion = '23.4.0'
     String mlKitVersion = '17.0.4'
     String roomVersion = "2.6.1"
     String espressoVersion = '3.5.1'
     String serialization_version = '1.6.2'
-    String metricsVersion = '2.1'
+    String metricsVersion = '4.2.25'
```

Comment:
This version does not exist. Where did this come from?

Reply:
https://mvnrepository.com/artifact/io.dropwizard.metrics/metrics-core

Reply:
?? we don't use `io.dropwizard.metrics`.